src/test/java/PokeLifeBotByMetin/tests/LogInTest.java

```diff
@@ -1,26 +1,13 @@
 package PokeLifeBotByMetin.tests;
 
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.PageFactory;
 import org.testng.annotations.Test;
+import PokeLifeBotByMetin.pages.LogInPage;
 
 public class LogInTest extends BaseTest {
-
-    @FindBy(name = "login")
-    private WebElement loginInput;
-
-    @FindBy(name = "haslo")
-    private WebElement passInput;
-
-    @FindBy(xpath = "//button[@class='btn btn-primary']")
-    private WebElement logInButton;
-
     @Test
     public void logInTest()  {
-        PageFactory.initElements(driver,this);
-        loginInput.sendKeys("lol");
-        passInput.sendKeys("lol123");
-        logInButton.click();
+        LogInPage logInPage = new LogInPage(driver);
+        logInPage.login("lol","lol");
     }
 }
+//branch2.2
```

Comment:
You can use "<version>1.0-SNAPSHOT</version>" for versioning. Before PR delete all working-comments.